integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -492,6 +537,38 @@ fn test_resharding_v3() {
     test_resharding_v3_base(TestReshardingParametersBuilder::default().build());
 }
 
+// Takes a sequence of shard ids to track in consecutive epochs,
+// repeats the last element `TESTLOOP_NUM_EPOCHS_TO_WAIT` times,
+// and maps each element: |id| -> vec![ShardId(id)], to the format required by `TrackedShardSchedule`.
+fn shard_sequence_to_schedule(mut shard_sequence: Vec<u64>) -> Vec<Vec<ShardId>> {
+    shard_sequence.extend(
+        std::iter::repeat(*shard_sequence.last().unwrap())
+            .take(TESTLOOP_NUM_EPOCHS_TO_WAIT as usize),
+    );
+    shard_sequence.iter().map(|shard_id| vec![ShardId::new(*shard_id)]).collect()
+}
+
+#[test]
+// TODO(resharding): fix nearcore and un-ignore this test
+#[ignore]
+fn test_resharding_v3_state_cleanup() {
+    // Track parent shard (6) before resharding, child shard (7) after resharding, and then an unrelated shard (5) forever.
```

Comment:
Are 6, 7 and 5 shard ids?

The parent shard UID is usually `s2.v3`, and the children `s3.v3` and `s4.v3`. Should we rather use 2, 4 and 1? Or am I missing something here?

Reply:
It's usually a bad idea to hardcode shard ids. It would be better to obtain those from the shard layout / test setup.

Reply:
fixed in 66f279e879863ae6b1144cdd6ba70c595c62cf6c